### DISCUSSION TITLE
如何在一个项目中分别指定构建32和64位的target

### DISCUSSION THREAD

**Discussion by Woodykaixa** [xmake-io/xmake#1922] [Q&A]
我正在编写一个调试器项目，在测试时需要分别测试32位和64位PE文件的解析。请问xmake是否可以实现在target中指定以32位或64位模式编译目标？

**Comment by waruqi** [comment]
对特定设置 set_arch() 具体看文档

**Reply by Woodykaixa** [reply]
谢谢，刚刚试过以后发现binary模式确实是生成了32位pe文件，但是shared模式下，链接器链接出来仍然是64位的

**Reply by Woodykaixa** [reply]
不好意思 最后发现是因为生成的文件在x86目录下，我打开的是x64目录下的旧版本文件
